examples/nextjs/src/app/[[...slug]]/page.js

```diff
@@ -4,45 +4,64 @@ import { MyPage } from "@/components/my-page";
 const client = dotcmsClient.init({
     dotcmsUrl: process.env.NEXT_PUBLIC_DOTCMS_HOST,
     authToken: process.env.DOTCMS_AUTH_TOKEN,
-    siteId: '59bb8831-6706-4589-9ca0-ff74016e02b2',
+    siteId: "59bb8831-6706-4589-9ca0-ff74016e02b2",
     requestOptions: {
         // In production you might want to deal with this differently
-        cache: 'no-cache'
-    }
+        cache: "no-cache",
+    },
 });
 
 export async function generateMetadata({ params, searchParams }) {
-
     const requestData = {
-        path: params?.slug ? params.slug.join('/') : 'index',
+        path: params?.slug ? params.slug.join("/") : "index",
         language_id: searchParams.language_id,
-        'com.dotmarketing.persona.id': searchParams['com.dotmarketing.persona.id'] || '',
+        "com.dotmarketing.persona.id":
+            searchParams["com.dotmarketing.persona.id"] || "",
         mode: searchParams.mode,
-        variantName: searchParams['variantName']
+        variantName: searchParams["variantName"],
     };
 
     const data = await client.page.get(requestData);
 
     return {
-        title: data.entity.page.friendlyName || data.entity.page.title
+        title: data.entity.page.friendlyName || data.entity.page.title,
     };
 }
 
 export default async function Home({ searchParams, params }) {
     const requestData = {
-        path: params?.slug ? params.slug.join('/') : 'index',
+        path: params?.slug ? params.slug.join("/") : "index",
         language_id: searchParams.language_id,
-        'com.dotmarketing.persona.id': searchParams['com.dotmarketing.persona.id'] || '',
+        "com.dotmarketing.persona.id":
+            searchParams["com.dotmarketing.persona.id"] || "",
         mode: searchParams.mode,
-        variantName: searchParams['variantName']
+        variantName: searchParams["variantName"],
     };
 
     const data = await client.page.get(requestData);
 
+    const content = await client.content
+        .getCollection("Blog")
+        // .draft(true)
+        .depth(2)
+        // .language(2)
+        // .render(true)
+        .sortBy([
+            {
+                field: "modDate",
+                order: "asc",
+            },
+        ])
+        .fetch();
```

Comment:
```suggestion
        .then(res => res.contentlets);
```

Reply:
Not sure of what we want to achieve. Should we get rid of `fetch` method?